README.md

```diff
@@ -13,34 +13,44 @@ This project is licensed under <a href="http://opensource.org/licenses/NPOSL-3.0
 https://github.com/jhundal/src/blob/master/bin/images/pvacseq-code.jpg
 
 ## Installation Instructions
-pVAC-Seq could either be used via cloning the git repo or by downloading the tarball.
-We strongly recommend the users to access it via git repo since updates are pushed before they make it to the stable release.
+pVAC-Seq could either be used via pip, cloning the git repo or by downloading the tarball.
+We strongly recommend the users to access it via pip as it will automatically fetch the most recent version, and automatically install the program for you
 
- 1. <b> OPTION 1 : Clone pVAC-Seq git repository: </b>
+ 1. <b> OPTION 1: Install pVAC-Seq from PIP: </b>
+        Make sure you have pip installed first.  pip is generally included in python distributions, but may need to be upgraded first.  See these instructions for installing or upgrading pip: https://packaging.python.org/en/latest/installing/#install-pip-setuptools-and-wheel
+
+  After you have pip installed, type the following command on your Terminal(for Mac and Linux users) or the command prompt (for Windows users): `pip install pvacseq`
+
+  pip will fetch and install pVAC-Seq and its dependencies for you.  After installing, you can run `pvacseq` directly from the Terminal/command prompt
+
+ 2. <b> OPTION 2 : Clone pVAC-Seq git repository: </b>
         Type the following command on your Terminal(for Mac and Linux users) or the command prompt (for Windows users):
-   `git clone git@github.com:griffithlab/pVAC-Seq.git  ` 
-  
+   `git clone git@github.com:griffithlab/pVAC-Seq.git  `
+
+  After cloning the repository you can either run the pvacseq scripts directly using: `python pvacseq/pvacseq.py` or by installing the script from the repo using `python setup.py install` which will enable you to run `pvacseq` directly from a Terminal/command prompt
+
  For more information, follow instructions for cloning a git repo :                               				  https://help.github.com/articles/cloning-a-repository/
 
- 2. <b> OPTION 2 : Download and decompress the tar ball</b>: 
+ 3. <b> OPTION 3 : Download and decompress the tar ball</b>:
 
- * `wget https://github.com/griffithlab/pVAC-Seq/archive/v1.0.0.tar.gz -O pVAC-Seq-1.0.0.tar.gz` 
+ * `wget https://github.com/griffithlab/pVAC-Seq/archive/v1.0.0.tar.gz -O pVAC-Seq-1.0.0.tar.gz`
  * `tar -xzf pVAC-Seq-1.0.0.tar.gz`
  * Make sure you see the following directories:
```

Comment:
Seems like having this list of files/dirs here is just going to be a pain, as it'll need to be continuously updated. I'd drop it.